OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -121,7 +121,11 @@ public void Mount(string name, string explicitName = null)
 					modPackages.Add(package);
 				}
 				else
+				{
 					package = OpenPackage(name);
+					if (package == null)
+						throw new InvalidOperationException("Could not open package '{0}', there is no support for this format.".F(name));
```

Comment:
Can you please change this to ", file not found or its format is not supported" to avoid misleading people if the file is missing?